### ISSUE TITLE
Error while submitting the invoice

### ISSUE THREAD

**Issue by halitsavafoglu** [checkout/checkout-magento2-plugin#385]
I have an error while using your extension version: 2.2.6 with Magento 2.2.3 and PHP 7.0, When I do Submit the Invoice throw Magento admin panel for any order that been paid with your payment method!

**We can't save the invoice right now.**

and the error inside "exception.log" file is:

[2020-11-04 06:27:09] main.CRITICAL: The endpoint did not accept the request. (Code: 403) {"exception":"[object] (Checkout\\Library\\Exceptions\\CheckoutHttpException(code: 403): The endpoint did not accept the request. (Code: 403) at /vendor/checkout/checkout-sdk-php/src/Library/HttpHandler.php:327)"} []

**Comment by saravanan0501** [comment]
Any fixes on this issue?? I'm getting the same!

**Comment by luke-fenn-cko** [comment]
Hello,

We have released a new version of the plugin (v 2.2.8) which includes improvements to the TransactionHandlerService.php that we believe was causing this issue. 
If this issue still persists after updating the plugin, please try increasing the Capture Time setting **Configuration -> Sales -> Payment Methods -> Checkout.com -> Configuration -> Global Settings -> Capture Time** in increments of 0.01 until this no longer happens.
Then, if this issue continues please try disabling the Webhook Table in the same area of admin as above.
I look forward to hearing back if this has fixed your error.

Many Thanks,
Luke

**Comment by halitsavafoglu** [comment]
When I did try to upgrade to the new version **2.2.8**, I faced this error while running **bin/magento setup:upgrade** command:

Updating modules:
Area code not set: Area code must be set before starting a session.


  [Magento\Framework\Exception\SessionException]
  Area code not set: Area code must be set before starting a session.



  [Magento\Framework\Exception\LocalizedException]
  Area code is not set

so I was forced to downgrade back again to version 2.2.6 to make the website work back again!
